.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+matrix:
+  include:
+  - python: 3.6
+    env:
+      - LABEL=docker_unit_tests
+    services:
+      - docker
+    script:
+      - docker build -t mozilla/vautomator-standalone .
+      - docker run -it mozilla/vautomator-standalone python3 -m pytest ./tests/
```

Comment:
This is already in master, not sure why it's still hanging around, possible branch source issue is my guess.